### ISSUE TITLE
`mouseover` selector not working, affecting examples such as Multi-Line Tooltip / Highlight

### ISSUE THREAD

**Issue by schlafel** [altair-viz/altair#3257]
# Problem statement
I noticed that with probably recent versions of browsers that Multi-Line-Highlights/Tooltips are not displayed anymore in html. 

This is also the case for the example page of the module:
[Multi-Line Highlight on documentation page](https://altair-viz.github.io/gallery/multiline_highlight.html).

![image](https://github.com/altair-viz/altair/assets/68279686/d2d54577-5378-4c43-93d2-b0566d0774b7)
If you open the graph in the vega editor, it works as expected:
![image](https://github.com/altair-viz/altair/assets/68279686/2e83cd4c-98a8-4e82-b90f-e98e953baa6a)

Am I the only one facing this problem? 

I was facing the same issue, when i was opening old file (that has worked before)

# Steps to reproduce: 

- Copy code from [Multi-Line Highlight on documentation page](https://altair-viz.github.io/gallery/multiline_highlight.html). and 
- Run it with Altair v5.1.2

**Comment by mattijn** [comment]
Thanks for raising @schlafel, I was about to say that I cannot reproduce the issue you are describing, but that was in safari. When trying the [multiline-tooltip](https://altair-viz.github.io/gallery/multiline_tooltip.html) and [multiline-highlight](https://altair-viz.github.io/gallery/multiline_highlight.html) example in google chrome I can indeed reproduce the issue you are describing. I personally do not think we should change the predicate to `mousemove` since `mouseover` should just be supported in chrome ([link](https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseover_event#browser_compatibility)). Maybe others can chime in with advice.

**Comment by schlafel** [comment]
I agree that `mouseover ` is more precise in this context. But since it was not working for Google Chrome as well as Microsoft Edge i had to come up with another solution.... 

But would be indeed interesting to have more details regarding this issue why it is currently not working in Chrome/Edge...

**Comment by mattijn** [comment]
I understand your situation, since it is functioning correctly within the vega-editor using the latest VL-version it is probably best that we upgrade Altair as well to the latest version of Vega-Lite.

**Comment by jbloom** [comment]
I am also having this issue, and for me it fails in both Chrome and Safari browsers. This also affects lots of similar plots I was making that worked until very recently.

I'm not sure what changed, but these plots worked for me no later than a week ago.

**Comment by mattijn** [comment]
A few hours ago this issue was raised at the Vega repository, https://github.com/vega/vega/issues/3825.
Currently Altair is loading the latest Vega v5 version, in this case v5.26. It seems that is the root of the cause..

**Comment by jheer** [comment]
Is it possible to instruct Altair which Vega version to load? If so, pegging the version to 5.25 might be a useful fix until a patched version of Vega is released.

**Comment by mattijn** [comment]
Since the described behaviour only affects the `canvas` rendering (default in altair) it is probably better to render as `svg` instead for the time being using `alt.renderers.set_embed_options(renderer='svg')`

Example spec:
```python
import altair as alt
from vega_datasets import data

source = data.cars()
nearest = alt.selection_point(on='mouseover', empty=False)

chart = alt.Chart(source).mark_circle(size=60).encode(
    x='Horsepower',
    y='Miles_per_Gallon',
    color='Origin',
    opacity=alt.condition(nearest, alt.value(1), alt.value(0.1)),
    size=alt.condition(nearest, alt.value(250), alt.value(50)),    
).add_params(nearest)

# option 1
# force the renderer to use `svg` instead of `canvas` as default
alt.renderers.set_embed_options(renderer='svg')
chart
```

<img src="https://github.com/altair-viz/altair/assets/5186265/c9cb7a2f-9107-434d-b471-f2060efe88b5" width="300"/>

Manually defining version 5.25 of Vega within Altair is not as easy as one might think. Since it depends on behaviour in the python kernel and on browser caching mechanisms. 

One could/should be able to do it as such, but I was not able successful rendering it:

```python
# reset to `canvas` if option 1 is tried first
alt.renderers.set_embed_options(renderer='canvas')

# option 2
# caution, this only works if other vega versions are not cached in the browser (very likely)
# create a custom HTMLRenderer with manual defined versions
custom_html_renderer = alt.display.HTMLRenderer(
    mode="vega-lite",
    template="universal",
    vega_version="5.25",  # default is alt.VEGA_VERSION
    vegaembed_version=alt.VEGAEMBED_VERSION,
    vegalite_version=alt.VEGALITE_VERSION,
)

# register the custom html renderer and enable it
alt.renderers.register('custom_html_renderer', custom_html_renderer)
alt.renderers.enable('custom_html_renderer')

chart  # no success
```

**Comment by jheer** [comment]
FYI I have a fix underway (vega/vega#3826) that I hope to merge and release soon.

**Comment by jheer** [comment]
Update: I just released Vega v5.26.1 with a fix.

**Comment by mattijn** [comment]
Thanks for the quick fix @jheer! You saved the Internet!
Thanks for raising the issue @schlafel! If it is right the newest Vega version is automatically loaded, if it is not the case and if it is urgent you might need to empty your browser cache.
